unit/ctest_mirror.c

```diff
@@ -0,0 +1,316 @@
+#include <math.h>
```

Comment:
can you rename this unit test so it's more descriptive? what about the mirror is it testing?
also, add comments in the test describing what it's testing

Reply:
ctest_gk_geometry_mirror.c